modules/web/js/ballerina/ast/ballerina-ast-root.js

```diff
@@ -176,12 +176,27 @@ class BallerinaASTRoot extends ASTNode {
     }
 
     /**
+     * Adds an import from a json object
+     * @param {Object} jsonNode json object representing the import
+     */
+    addImportFromJson(jsonNode) {
+        const importNode = this.getFactory().createFromJson(jsonNode);
+        if (!this.getFactory().isImportDeclaration(importNode)) {
+        // only imports can added at global level
+            return;
+        }
+        importNode.initFromJson(jsonNode);
+        this.addImport(importNode);
+    }
+
+
+    /**
      * Adds new import declaration.
      * @param {ImportDeclaration} importDeclaration - New import declaration.
```

Comment:
Changed the identifier name as importDeclaration to be consistent in commit 3680b229404cacc4c299e52adc1692db2c581a16